src/eko/strong_coupling.py

```diff
@@ -306,6 +318,7 @@ def a_s(self, scale_to, fact_scale=None, nf_to=None):
         is_downward_path = False
         if len(path) > 1:
             is_downward_path = path[1].nf < path[0].nf
+        shift = 3 if not is_downward_path else 4
```

Comment:
We learnt the lesson!

Reply:
Indeed, but I'm not completely sure about it. And I don't know actually how to test it properly at the PDF level (APFEL is not doing backward, so we should evolve forward and see if eko is able to go back...), probably it deserves a dedicated section in `benchmark_strong_coupling`...

Reply:
It looks consistent with the rest of the backward implementation, whenever you match you use the `nf` in the lower patch.